### PR TITLE
St/events

### PR DESCRIPTION
**Pull request by saratnt**

### CODE REVIEW

**Review by JcBernack — APPROVED** [review]
libevm/go parts look good to me

File: sdk/src/main/scala/com/horizen/account/events/AddNewForgerStakeEvent.scala [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+import com.horizen.proposition.MCPublicKeyHashProposition
+import com.horizen.utils.BytesUtils
+import org.web3j.abi.datatypes.Address
+import org.web3j.abi.datatypes.generated.{Bytes20, Bytes32, Uint256, Uint32}
+
+import java.math.BigInteger
+import scala.annotation.meta.getter
+
+
+case class AddNewForgerStakeEvent(@(Parameter@getter)(1) @(Indexed@getter) from: Address,
```
Comment (i-Alex) [inline sdk/src/main/scala/com/horizen/account/events/AddNewForgerStakeEvent.scala:14]: rename to "DelegateForgerStake"

File: sdk/src/main/scala/com/horizen/account/events/RemoveForgerStakeEvent.scala [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+import com.horizen.account.event.{Indexed, Parameter}
+import com.horizen.account.proposition.AddressProposition
+import com.horizen.utils.BytesUtils
+import org.web3j.abi.datatypes.Address
+import org.web3j.abi.datatypes.generated.Bytes32
+
+import scala.annotation.meta.getter
+
+
+case class RemoveForgerStakeEvent(@(Parameter@getter)(1) @(Indexed@getter) owner: Address,
```
Comment (i-Alex) [inline sdk/src/main/scala/com/horizen/account/events/RemoveForgerStakeEvent.scala:12]: rename to "WithdrawForgerStake"

**Review by i-Alex — CHANGES_REQUESTED** [review]
Good!

File: sdk/src/main/scala/com/horizen/account/events/RemoveForgerStakeEvent.scala (hunk 2 above)
Comment (saratnt) [reply sdk/src/main/scala/com/horizen/account/events/RemoveForgerStakeEvent.scala]: Done

File: sdk/src/main/scala/com/horizen/account/events/AddNewForgerStakeEvent.scala (hunk 1 above)
Comment (saratnt) [reply sdk/src/main/scala/com/horizen/account/events/AddNewForgerStakeEvent.scala]: Done

File: sdk/src/main/scala/com/horizen/account/events/AddWithdrawalRequestEvent.scala [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+import com.horizen.proposition.MCPublicKeyHashProposition
+import com.horizen.utils.BytesUtils
+import org.web3j.abi.datatypes.Address
+import org.web3j.abi.datatypes.generated.{Bytes20, Uint256, Uint32}
+
+import java.math.BigInteger
+import scala.annotation.meta.getter
+
+
+case class AddWithdrawalRequestEvent(@(Parameter@getter)(1) @(Indexed@getter) from: Address,
```
[the remark this answers is not in the document]
Comment (saratnt) [reply sdk/src/main/scala/com/horizen/account/events/AddWithdrawalRequestEvent.scala]: Done

**Review by i-Alex — APPROVED** [review]

**Pull request merged**